lib/src/main/java/com/tubitv/media/activities/TubiPlayerActivity.java

```diff
@@ -157,7 +158,7 @@ protected void initLayout() {
     private void setCaption(boolean isOn) {
         if (mediaModel != null && mediaModel.getSubtitlesUrl() != null && mTubiPlayerView != null
                 && mTubiPlayerView.getControlView() != null) {
-            //            mTubiPlayerView.getControlView().checkSubtitleIcon(isOn);
+            ((UserController) mTubiPlayerView.getPlayerController()).isSubtitleEnabled.set(isOn);
             mTubiPlayerView.getSubtitleView().setVisibility(isOn ? View.VISIBLE : View.GONE);
```

Comment:
Can we use UserController.triggerSubtitlesToggle() directly for reusability, and delete the line 162